framework/db/Connection.php

```diff
@@ -618,9 +625,10 @@ protected function initConnection()
      */
     public function createCommand($sql = null, $params = [])
     {
-        $command = new Command([
+        $command = \Yii::createObject([
```

Comment:
> What is the argument for \Yii::createObject() if we have Instance::ensure()?

I don't know the original idea, but I see that `Instance::ensure` was introduced after `Yii::createObject()` and after `BaseMailer::createMessage()`. It seems to be designed for the cases, we are talking about, but is not used where it's needed. @yiisoft/core-developers